knowledge/science/botany/jasminum/qna.yml

```diff
@@ -0,0 +1,19 @@
+created_by: ghloeffler
+domain: flowers
+seed_examples:
+- answer: Jasminum sambac is a species of jasmine native to tropical asia.
+  question: What is Jasminum sambac?
+- answer: | Jasminum sambac is an evergreen vine or shrub that grows up to 9.8 ft tall. The leaves are ovate 1.6 to 4.9 inches long and .79 to 2.95 inches wide. 
```

Comment:
Thank you for your submission; taking a look at your `qna.yaml`, we noticed that you have some issues with the multiline `yaml` syntax.

Can you take a look at: https://yaml-multiline.info/ and fix up the file? Thanks!